### ISSUE TITLE
Unexpected token ...

### ISSUE THREAD

**Issue by gregoirepuget** [betsol/gulp-require-tasks#19]
Hi,

I've got this error :
/node_modules/gulp-require-tasks/index.js:93
taskNameParts.push(...pathInfo.dir.split(path.sep));
                                   ^^^

SyntaxError: Unexpected token ...
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:373:25)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)

Do you know what can be the error ?

Thank you

**Comment by slavafomin** [comment]
Hello! Thank you for your interest in this module!

What version of Node are you using?

**Comment by slavafomin** [comment]
@gregoirepuget Looks like we've accidentally used spread operator from ES6 in one of the recent PRs. I've studied the node version tables and it looks like the latest maintained version is `4.8`. I've updated the code to ensure backward compatibility for this version. Feel free to check out the new `1.1.1` release, it should work fine.

However, if you are using version prior to `4.8` I will highly advise you to update to at least `4.8`, because previous versions are no longer maintained.
